pkg/application/create.go

```diff
@@ -59,17 +62,24 @@ func checkoutNewBranch(localRepo *git.LocalRepository, branchName string) error
 	if currentBranch != git.MainBranch {
 		if err := localRepo.CheckoutBranch(&git.CheckoutOp{
 			BranchName: git.MainBranch,
+			DryRun:     false,
```

Comment:
Is there a reason for ignoring passed DryRun here?

Reply:
No, just missed it, thanks!